packages/runtime-js/src/main/kotlin/elide/runtime/plugins/js/Extensions.kt

```diff
@@ -0,0 +1,28 @@
+package elide.runtime.plugins.js
+
+import org.graalvm.polyglot.Source
+import elide.runtime.core.DelicateElideApi
+import elide.runtime.core.PolyglotContext
+import elide.runtime.core.PolyglotValue
+import elide.runtime.core.evaluate
+
+/**
+ * Execute the given JavaScript [source] code, returning the result. This is equivalent to calling
+ * [PolyglotContext.evaluate] and selecting [JavaScript] as source language.
+ *
+ * @param source The JavaScript source code to be executed.
+ * @param esm Whether to treat the [source] as an ESM module. If false, the code is evaluted as CommonJS source.
+ * @return The result of the invocation. If [esm] is `true`, an object is returned, with exported values as members.
+ */
+@DelicateElideApi public fun PolyglotContext.javaScript(
```

Comment:
nit, can it by `javascript`? so it lines up with `python`/`ruby` eventually

Reply:
good point, will update